source/03-components/modal/modal.es6.js

```diff
@@ -0,0 +1,112 @@
+import Drupal from 'drupal';
+import once from 'once';
+import KEYCODE from '../../00-config/_KEYCODE.es6';
```

Comment:
Do we still need this? Can’t we rely on `key`?

Reply:
Removed references to _KEYCODE.es6